test/models/development_test.rb

```diff
@@ -203,19 +204,81 @@ def development
 
   test "nearby developments" do
     far_dev = developments(:one)
-    far_dev.latitude = 40.000000
+    far_dev.latitude  =  40.000000
     far_dev.longitude = -77.000000
     far_dev.save
 
     close_dev = developments(:two)
-    close_dev.latitude = 39.010000
+    close_dev.latitude  =  39.010000
     close_dev.longitude = -75.990000
     close_dev.save
 
     close_devs = Development.close_to(39.000000, -76.000000).load
 
-    assert_equal 1,          close_devs.size
+    assert_equal 1,         close_devs.size
     assert_equal close_dev, close_devs.first
   end
 
+  test "range scopes, dates" do
+    range_scopes = %i( created_at updated_at )
+    range_scopes.each {|scope| assert_respond_to Development, scope }
+  end
+
+  test "range scopes, float and integer" do
+    ranged_scopes.each {|scope| assert_respond_to Development, scope }
+  end
+
+
+  test "boolean scopes" do
+    boolean_scopes.each {|s| assert_respond_to Development, s }
+  end
+
+  test "boolean scope definition" do
+    assert_equal 1, Development.hidden(true).count
+    assert_equal 1, Development.hidden.count
+    assert_equal 1, Development.hidden(false).count
+    assert_equal Development.hidden(true), Development.hidden
+    refute_equal Development.hidden(true), Development.hidden(false)
+  end
+
+  # These tests and the params methods could stand some refactoring
+  test "periscope" do
```

Comment:
There's a better way to write these tests. I will do that.

It won't be ideal, because ideally this functionality would be in its own gem, but it's in `/lib` for now.